"""add column error

Revision ID: 3842910ac6d2
Revises: 3fe3ba525bed
Create Date: 2025-04-01 02:15:00.596604

"""
from typing import Sequence, Union

from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision: str = '3842910ac6d2'
down_revision: Union[str, None] = '3fe3ba525bed'
branch_labels: Union[str, Sequence[str], None] = None
depends_on: Union[str, Sequence[str], None] = None


def upgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('fact_api_call', sa.Column('api_error_message', sa.String(), nullable=True), schema='game_xgame_drama_max_schema')
    # ### end Alembic commands ###


def downgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_column('fact_api_call', 'api_error_message', schema='game_xgame_drama_max_schema')
    # ### end Alembic commands ###
